package.json

```diff
@@ -563,6 +563,7 @@
     "crypto-browserify": "^3.12.0",
     "css-loader": "^6.10.0",
     "css-to-xpath": "^0.1.0",
+    "cssnano": "^7.0.7",
```

Comment:
Adding [cssnano](https://github.com/cssnano/cssnano) to minify css in build pipeline